src/debug.c

```diff
@@ -1020,7 +1020,7 @@ void _serverAssertPrintClientInfo(const client *c) {
 
     bugReportStart();
     serverLog(LL_WARNING, "=== ASSERTION FAILED CLIENT CONTEXT ===");
-    serverLog(LL_WARNING, "client->flags = %llu", (unsigned long long)c->flags);
+    // serverLog(LL_WARNING, "client->flags = %llu", (unsigned long long)c->flags);
```

Comment:
This should still work, is there a reason this was dropped?

Reply:
I made a mistake in writing. It has been corrected.